brave/brave-core
# Toggle active tab's audio mute with `ctrl + m`

fix https://github.com/brave/brave-browser/issues/26994


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Load youtube.com and play any clip
2. Check `ctrl + m` toggles audio mute
